src/con_duct/suite/ls.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+from collections import OrderedDict
+import glob
+import json
+from typing import List
+import pyout  # type: ignore
+import yaml
+from con_duct.__main__ import DUCT_OUTPUT_PREFIX, SummaryFormatter
+
+VALUE_TRANSFORMATION_MAP = {
+    "exit_code": "{value!E}",
+    "wall_clock_time": "{value:.3f} sec",
+    "peak_rss": "{value!S}",
+}
```

Comment:
@yarikoptic if we use the custom formatter like this, the user can't provide their own format string, so its a bit less flexible than we did for regular duct output, but I think its a whole lot cleaner/dryer. And the user experience seems better too-- ie, provide an ordered list of fields that you want, rather than a format string that requires knowledge of how the formatter works.

If you like this approach too, I'll fill out this map with all the fields, and perhaps refactor regular duct output to function this way.

Reply:
get some way done, let us try and then lets discuss and potentially improve.  

Eventually I would expect us to be able to extend list of relevant attributes to be visualized in `ls` based on user's inputs.  Might even be some kind of a "run a regex across stdout or stderr to get information of interest to be printed in each row" to support some custom extraction but for that I do not have immediate use case hence just a wild idea (must **not** be yet implemented anyhow).